### ISSUE TITLE
Minor quibble - docs need to be more specific on time zone

### ISSUE THREAD

**Issue by patientplatypus** [node-schedule/node-schedule#359]
Hi.

I've noticed while using your application that the docs say to use time, but I assumed that this needs to be based on GMT, so I offset my time accordingly. It would be nice to have a slight change to the documentation online to let users know that it is set up to use your own local time by default.

Thank you.

**Comment by santigimeno** [comment]
Agreed. Would you consider sending a PR to improve the documentation? Thanks!

**Comment by ThoseGuysInTown** [comment]
Yeah this would help a lot, or some way to specifically set the timezone for each job. Thanks!

**Comment by vb-vasudev** [comment]
Hey guys, just to clarify - is there a way to set a specific time zone?

**Comment by FabricioPavano** [comment]
it's not clear to me, does it use local time or GMT by default?

**Comment by olaoluwa-98** [comment]
@FabricioPavano it uses the machine's local time by default
